detection_rules/etc/version.lock.json

```diff
@@ -315,9 +315,16 @@
       }
     },
     "rule_name": "Remote Desktop Enabled in Windows Firewall by Netsh",
-    "sha256": "bf0429e76fb9c1db6f809649d079add564548ab3be0cde7b59b0927794bb0535",
+    "sha256": "6abac6bdb953f843131c74427b2a2d7868be53e3d1ece8e42dbd69f7bd457896",
     "type": "eql",
-    "version": 100
+    "version": 101
+  },
+  "07b5f85a-240f-11ed-b3d9-f661ea17fbce": {
+    "min_stack_version": "8.3",
+    "rule_name": "Google Drive Ownership Transferred via Google Workspace",
+    "sha256": "f8e05498e63a3fb10621fd91713a7f0995aaf07a9eb6fd5ef73b62c7a81458f6",
+    "type": "query",
+    "version": 1
```

Comment:
Do we leave rule versions at 1? Or during the fork, it auto-bumps the first previous entry by 100?